From 715db732f9d82ca2d13554a2ae88384e8e017584 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Thu, 12 Sep 2024 08:09:56 +0200
Subject: [PATCH 1190/1695] core: panic: add missing inclusion of keep.h

Add missing inclusion of keep.h in panic.c.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 4ab5a8ec33a4bb3196c4b8b7479d161c215f7ea2 core: panic: add missing inclusion of keep.h]
Change-Id: I180ff5bed345ffa15e96bfde7212ba34903e959b
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/403895
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/kernel/panic.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/kernel/panic.c b/core/kernel/panic.c
index 5851c2c73..2950466f8 100644
--- a/core/kernel/panic.c
+++ b/core/kernel/panic.c
@@ -5,6 +5,7 @@
  */
 
 #include <initcall.h>
+#include <keep.h>
 #include <kernel/cache_helpers.h>
 #include <kernel/interrupt.h>
 #include <kernel/misc.h>
-- 
2.47.2

